### DISCUSSION TITLE
number of connections

### DISCUSSION THREAD

**Discussion by liu-yihong-1** [prometheus/prometheus#12108] [Q&A]
### Proposal

Prometheus does not monitor its own number of connections

**Comment by roidelapluie** [comment]
Can you elaborate a bit more on what you mean by "Prometheus does not monitor its own number of connections"? Specifically, what are you referring to? It would be great if you could provide more details, so that we can better understand your proposal. Thanks!

**Reply by liu-yihong-1** [reply]
The default parameter of Prometheus -- web. max connections=512, but we actually don't know how many have been used. If there is a lot of concurrency, many error messages will be returned
